### ISSUE TITLE
Text Font size for legend and axis metrics/labels.

### ISSUE THREAD

**Issue by Copter64** [grafana/grafana#12628]
Read before posting: 

- Questions should be posted to https://community.grafana.com. Please search there and here on GitHub for similar issues before creating a new issue. 
- Checkout FAQ: https://community.grafana.com/c/howto/faq
- Checkout How to troubleshoot metric query issues: https://community.grafana.com/t/how-to-troubleshoot-metric-query-issues/50

Please include this information:
What Grafana version are you using?
v 5.2.1

What datasource are you using?
PRTG

What OS are you running grafana on?
Ubuntu 16.04.4 LTS

What did you do?
Setup Panels based on different PRTG Sensors

What was the expected result?
Want the text size for the legend and x and y axis labels to be larger

What happened instead?
On a 4k screen that we are using, the text shows up really small and we haven't found a way to scale the text up without losing space from our graphs.

If related to metric query / data viz:

**Comment by daniellee** [comment]
Can you show a screenshot of what it looks like on a 4k screen?

**Comment by torkelo** [comment]
using regular browser zoom (CTRL+) usually works really well for 4k screens. Granted it will make other stuff bigger as well but that is usually good as it keep the design in sync.

**Comment by Copter64** [comment]
![image](https://user-images.githubusercontent.com/24832022/42817939-94686b4a-899d-11e8-8504-5d3ead120069.png)

**Comment by Copter64** [comment]
We were able to modify the font for pings with the panel JSON.  Is there a similar syntax for the legend,x and y axis fonts?

**Comment by Copter64** [comment]
Here is our Ping example. But I wasn't sure where font changes would need to be applied for the Graph's JSON.

{
  "cacheTimeout": null,
  "colorBackground": false,
  "colorValue": false,
  "colors": [
    "#299c46",
    "rgba(237, 129, 40, 0.89)",
    "#d44a3a"
  ],
  "datasource": "rkv-prtg01",
  "format": "none",
  "gauge": {
    "maxValue": 100,
    "minValue": 0,
    "show": false,
    "thresholdLabels": false,
    "thresholdMarkers": true
  },
  "gridPos": {
    "h": 4,
    "w": 2,
    "x": 7,
    "y": 22
  },
  "id": 45,
  "interval": null,
  "links": [],
  "mappingType": 1,
  "mappingTypes": [
    {
      "name": "value to text",
      "value": 1
    },
    {
      "name": "range to text",
      "value": 2
    }
  ],
  "maxDataPoints": 100,
  "nullPointMode": "connected",
  "nullText": null,
  "postfix": " ms",
  "postfixFontSize": "150%",
  "prefix": "",
  "prefixFontSize": "50%",
  "rangeMaps": [
    {
      "from": "null",
      "text": "N/A",
      "to": "null"
    }
  ],
  "sparkline": {
    "fillColor": "rgba(31, 118, 189, 0.18)",
    "full": false,
    "lineColor": "rgb(31, 120, 193)",
    "show": true
  },
  "tableColumn": "",
  "targets": [
    {
      "channel": {
        "name": "RTT (Average)"
      },
      "device": {
        "name": ""
      },
      "errors": {},
      "functions": [],
      "group": {
        "name": ""
      },
      "options": {
        "includeDeviceName": false,
        "includeSensorName": false,
        "mode": {
          "filterProperty": {},
          "name": "Metrics",
          "textProperty": {},
          "textValueFrom": {},
          "value": 1
        }
      },
      "raw": {
        "queryString": "",
        "uri": ""
      },
      "refId": "A",
      "sensor": {
        "name": "QoS RoundTrip - RKV to IC"
      }
    }
  ],
  "thresholds": "",
  "title": "RKV to IC",
  "type": "singlestat",
  "valueFontSize": "150%",
  "valueMaps": [
    {
      "op": "=",
      "text": "N/A",
      "value": "null"
    }
  ],
  "valueName": "avg"
}

**Comment by Copter64** [comment]
What are the CSS attributes for the legends, x and y axis?  Are we able to apply CSS to those axis and legend?

**Comment by Copter64** [comment]
![4k screenshot](https://user-images.githubusercontent.com/24832022/42828154-14ae25ca-89b6-11e8-9fb3-4b82c1d07289.png)

**Comment by Copter64** [comment]
Yeah, I think we will try to zoom instead of there isn't another way.
